### PR TITLE
Fixes mech construction messages

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
spellcheck: Mech construction messages no longer incorrectly mention high-tier stock parts.
/:cl:

Many mech construction messages mention T3/T4 stock parts, which is flat out wrong. You can use any tier of stock part; the higher, the more EMP resistance/less power consumption.

### CODE REVIEW

**Comment by ninjanomnom** [comment]
Why aren't we just using the actual name of the thing being installed?

**Comment by 81Denton** [comment]
That makes more sense to be honest

**Comment by 81Denton** [comment]
Alright, I replaced most static stock part names with `[I.name]`. I have no clue how to reference stock parts that just got installed though.

File: code/game/mecha/mecha_construction_paths.dm [hunk 1]
```diff
@@ -198,7 +198,7 @@
 				user.visible_message("[user] removes the wiring from [parent].", "<span class='notice'>You remove the wiring from [parent].</span>")
 		if(5)
 			if(diff==FORWARD)
-				user.visible_message("[user] installs the central control module into [parent].", "<span class='notice'>You install the central computer mainboard into [parent].</span>")
+				user.visible_message("[user] installs the [I.name] into [parent].", "<span class='notice'>You install the [I.name] into [parent].</span>")
```
Comment (ninjanomnom) [inline code/game/mecha/mecha_construction_paths.dm]: Just needs to be `[I]` and I *think* the `the` before is unnecessary?

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/game/mecha/mecha_construction_paths.dm (hunk 1 above)
Comment (81Denton) [reply code/game/mecha/mecha_construction_paths.dm]: Oh yeah, `[I]` works. The extra "the" is neccessary, else we'd get messages like `You install Ripley Central Control module (Exosuit Board) into the Ripley chassis.`

**Review by SpaceManiac — CHANGES_REQUESTED** [review]
`the [I]` will generate lots of double-the, and if it doesn't now it will eventually. Should just be `installs [I] into [parent]`, and if you want the resulting sentence to include "the", you need `\improper` on the names if they aren't already lowercased.

**Comment by 81Denton** [comment]
Thanks for the explaination, I'll go remove the extra `the`.

File: code/game/mecha/mecha_construction_paths.dm [hunk 2]
```diff
@@ -494,27 +494,27 @@
...
 		if(11)
 			if(diff==FORWARD)
-				user.visible_message("[user] installs scanner module to [parent].", "<span class='notice'>You install scanner module to [parent].</span>")
+				user.visible_message("[user] installs [I] to [parent].", "<span class='notice'>You install [I] to [parent].</span>")
 			else
 				user.visible_message("[user] unfastens the weapon control module.", "<span class='notice'>You unfasten the weapon control module.</span>")
 		if(12)
 			if(diff==FORWARD)
-				user.visible_message("[user] secures the advanced scanner module.", "<span class='notice'>You secure the scanner module.</span>")
+				user.visible_message("[user] secures the scanner module.", "<span class='notice'>You secure the scanner module.</span>")
```
Comment (ninjanomnom) [inline code/game/mecha/mecha_construction_paths.dm]: Missed a few?

Comment (81Denton) [reply code/game/mecha/mecha_construction_paths.dm]: It looks like stock parts get stored in the chassis' `contents` once inserted. Do you know how I can access their name?

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**